data/json/monsters.json

```diff
@@ -6188,7 +6188,26 @@
     },
     "hp":80,
     "death_function":"EXPLODE",
-    "special_attacks":[["SMG", 2]],
+    "special_attacks":[
+        {
+            "type" : "gun",
+            "cooldown" : 2,
+            "gun_type" : "hk_mp5",
```

Comment:
This would be an ideal time to define `fake_smg` in `item/gun/fake.json` as discussed in #15227